aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -529,7 +531,7 @@ bool isFunctionalTensorIListRef(c10::IListRef<T> list) {
 }
 
 bool isFunctionalTensor(ITensorListRef list) {
-  return isFunctionalTensorIListRef(list);
+  return isFunctionalTensorIListRef(std::move(list));
```

Comment:
I don't understand how IListRef is movable at all currently, but this looks like the wrong fix -- isFunctionalTensorIListRef is not trying to take ownership

Reply:
Actually, taking const ref causes overload ambiguity interestingly enough.